src/plugin/featureaction/featureaction.js

```diff
@@ -191,12 +191,5 @@ plugin.im.action.feature.editEntry = function(entryType, opt_entry) {
  * @return {boolean} Whether to refresh the layer.
  */
 plugin.im.action.feature.shouldRefresh = function(entryType) {
```

Comment:
If this function is no longer needed, we should remove it entirely.

Reply:
Agreed. I wanted to run it by you before making the call to remove it. I wasn't sure if there was some good reason to keep the refresh in for some case.